### ISSUE TITLE
Don't use default scope `default_scope`

### ISSUE THREAD

**Issue by HeroicEric** [envoy/styleguides#8]

**Comment by warmwaffles** [comment]
Yep, Add it.

**Comment by mattharris5** [comment]
I don't think it should be a blanket prohibition. Default scopes need to be used with care, but there are times that it might make sense. Since we're using `uuid`'s as PK's in Deliveries, `created_at` becomes a much more sane way of default ordering result sets and it's reasonable for that to be the default sort. With `reorder` and `unscoped(:attribute)`, scopes can stay pretty manageable.
